scvi/inference/__init__.py

```diff
@@ -1,29 +1,26 @@
 from .posterior import Posterior
 from .trainer import Trainer
-from .inference import (
-    UnsupervisedTrainer,
-    AdapterTrainer,
-)
+from .inference import UnsupervisedTrainer, AdapterTrainer
 from .annotation import (
     JointSemiSupervisedTrainer,
     SemiSupervisedTrainer,
     AlternateSemiSupervisedTrainer,
-    ClassifierTrainer
+    ClassifierTrainer,
 )
-from .fish import TrainerFish
+from .jvae_trainer import JVAETrainer
 from .autotune import auto_tune_scvi_model, hyperopt_worker, launch_workers
 
 __all__ = [
-    'Trainer',
-    'Posterior',
-    'TrainerFish',
-    'UnsupervisedTrainer',
-    'AdapterTrainer',
-    'JointSemiSupervisedTrainer',
-    'SemiSupervisedTrainer',
-    'AlternateSemiSupervisedTrainer',
-    'ClassifierTrainer',
-    'auto_tune_scvi_model',
-    'hyperopt_worker',
-    'launch_workers',
+    "Trainer",
+    "Posterior",
+    "UnsupervisedTrainer",
+    "AdapterTrainer",
+    "JointSemiSupervisedTrainer",
+    "SemiSupervisedTrainer",
+    "AlternateSemiSupervisedTrainer",
+    "ClassifierTrainer",
+    "JVAETrainer",
+    "auto_tune_scvi_model",
+    "hyperopt_worker",
+    "launch_workers",
```

Comment:
Does the lint program (flake8) check for single quotes now? Otherwise it's going to be hard to maintain these coding standards.

Reply:
De are gradually introducing black which does!

Reply:
Nope it doesn't enforce that, but `black` does and its usage is recommended in `CONTRIBUTING.rst`. I let the formatter only affect this file because the git history is not very important for this file. Tell me if you think I should revert the changes.